gita/__main__.py

```diff
@@ -201,6 +201,7 @@ def f_clone(args: argparse.Namespace):
 
     repos, groups = io.parse_clone_config(args.clonee)
```

Comment:
it will be more readable if we rename them to `repos_to_clone`, `groups_to_clone`